### PR TITLE
Fix/config missing

### PR DESCRIPTION
**Pull request by max-wittig**

### CODE REVIEW

File: gitlab/config.py [hunk 1]
```diff
@@ -41,21 +41,37 @@ class GitlabConfigParser(object):
     def __init__(self, gitlab_id=None, config_files=None):
         self.gitlab_id = gitlab_id
         _files = config_files or _DEFAULT_FILES
+        file_exist = False
+        for file in _files:
+            if os.path.exists(file):
+                file_exist = True
+        if not file_exist:
+            exit(
```
Comment (42B) [inline gitlab/config.py]: `sys.exit` ?

File: gitlab/config.py [hunk 2]
```diff
@@ -41,21 +41,37 @@ class GitlabConfigParser(object):
...
         self._config = configparser.ConfigParser()
         self._config.read(_files)
 
         if self.gitlab_id is None:
             try:
                 self.gitlab_id = self._config.get('global', 'default')
             except Exception:
-                raise GitlabIDError("Impossible to get the gitlab id "
-                                    "(not specified in config file)")
+                exit(
```
Comment (42B) [inline gitlab/config.py]: `sys.exit`?

File: gitlab/config.py [hunk 3]
```diff
@@ -41,21 +41,37 @@ class GitlabConfigParser(object):
...
+                    "It should contains at least a [global] section, \n"
+                    "and a specific section for each GitLab server."
+                )
 
         try:
             self.url = self._config.get(self.gitlab_id, 'url')
         except Exception:
-            raise GitlabDataError("Impossible to get gitlab informations from "
-                                  "configuration (%s)" % self.gitlab_id)
+            exit(
```
Comment (42B) [inline gitlab/config.py]: `sys.exit`?

**Comment by max-wittig** [comment]
@gpocentek Thanks for the info. I will take care of this later

**Pull request merged**